issue_template.md

```diff
@@ -0,0 +1,29 @@
+# Bug: -Title-
```

Comment:
I think you can automatically set a few properties like labels by adding some yaml frontmatter to the file. Something like this:

```yaml
---
name: Bug
about: Use this template for reporting bugs in Thrift.Net.
labels: "type/bug"
---
```

There's some details about it here: https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/manually-creating-a-single-issue-template-for-your-repository

Should we maybe call the file `bug_template.md` as well?

Can I get you to move this to a `.github` directory as well, just so we put anything related to automating GitHub in the one place?